[ERROR] Invalid option in build() call: "watch"

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

1.4.0

### Current Behavior

Running `wagmi generate` throws the following error:

```
$ wagmi generate
✘ [ERROR] Invalid option in build() call: "watch"

   node_modules/esbuild/lib/main.js:255:12:
 255 │       throw new Error(`Invalid option ${where}: ${quote(key)}`);
```

### Expected Behavior

wagmi generate runs normally

### Steps To Reproduce

_No response_

### Link to Minimal Reproducible Example (StackBlitz, CodeSandbox, GitHub repo etc.)

_No response_

### Anything else?

Maybe it's related to https://github.com/wagmi-dev/wagmi/pull/2960

--- a/.changeset/kind-icons-tap.md
+++ b/.changeset/kind-icons-tap.md
@@ -1,0 +1,5 @@
+---
+"@wagmi/cli": patch
+---
+
+Fixed esbuild version

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -78,7 +78,7 @@
     "detect-package-manager": "^2.0.1",
     "dotenv": "^16.0.3",
     "dotenv-expand": "^10.0.0",
-    "esbuild": "0.19.0",
+    "esbuild": "0.16.17",
     "execa": "^6.1.0",
     "find-up": "^6.3.0",
     "fs-extra": "^10.1.0",

